api/container/containerv1/worker_pool.go

```diff
@@ -127,6 +129,16 @@ func (w *workerpool) ResizeWorkerPool(clusterNameOrID, workerPoolNameOrID string
 	return err
 }
 
+// UpdateLabelsWorkerPool calls the API to resize a worker with the labels option
+func (w *workerpool) UpdateLabelsWorkerPool(clusterNameOrID, workerPoolNameOrID, labels map[string]string, target ClusterTargetHeader) error {
+	requestBody := WorkerPoolPatchRequest{
+		State:  "labels",
```

Comment:
Is this required? I would think that state was only needed when the worker pool was actually resizing...?

Reply:
Yes, according to the API docs.    "To apply a custom label to the worker pool, set the "state" field to "labels" and fill in the "labels" section"
https://containers.cloud.ibm.com/global/swagger-global-api/#/clusters/PatchWorkerPool